webcompat/static/js/lib/bugform.js

```diff
@@ -142,7 +142,7 @@ function BugForm() {
     if (!location.search.search(/open=1/) && !location.search.search(/url=/)) {
       return;
     }
-    var urlParam = location.search.match(/url=(.+)/);
+    var urlParam = location.search.match(/url=([^&]*)/);
```

Comment:
Do you have a test for URLs like 
http://www.openstreetmap.org/search?query=paris#map=11/48.8591/2.2611&layers=C

If they are properly escaped, it will be ok, but if not:

```js
/* this is bogus already but let say */
url = 'http://www.example.com/?open=1&url=http://www.openstreetmap.org/search?query=paris#map=11/48.8591/2.2611&layers=C&src=FASTER'
url.match(/url=([^&]*)/)
/* will return 

Array [ "url=http://www.openstreetmap.org/search?query=paris#map=11/48.8591/2.2611", "http://www.openstreetmap.org/search?query=paris#map=11/48.8591/2.2611" ]
*/
```
which is not what we want.